engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -122,6 +122,7 @@ public boolean isLowerLayerVisible() {
 
     @Override
     public void onOpened() {
+    	super.onOpened();
```

Comment:
Eeeeevil tab character! We follow the space side of the Force :D

You should be able to configure your editor to prefer 4 spaces instead of a tab char :-)

(fixing as part of merge so no worries here)